libs/blocks/global-navigation/global-navigation.js

```diff
@@ -465,6 +465,11 @@ class Gnav {
     const profileData = await fetch(`https://${env.adobeIO}/profile`, { headers });
 
     if (profileData.status !== 200) {
+      lanaLog({
+        message: 'GNAV: decorateProfile has failed to fetch profile data',
+        e: `${profileData.statusText} url: ${profileData.url}`,
+        tags: 'errorType=warn,module=gnav',
```

Comment:
I think the `errorType=info` would be more appropriate in here. You can see the tags legend in [here](https://github.com/adobecom/milo/pull/2314). 
```suggestion
        tags: 'errorType=info,module=gnav',
```

Reply:
@robert-bogos 
Done.